lib/Dialect/IntelGpu/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(IR)
```

Comment:
@nbpatel, just wanted to point this out. Maybe clang-format issue. Might be better to use the clang-format to format.